src/Kinepolis/KinepolisService.php

```diff
@@ -152,6 +153,7 @@ private function process(ParsedMovie $parsedMovie, string $token): void
 
         if ($eventId === null) {
             $eventId = $this->createNewMovie($parsedMovie);
+            $commands[] = new Publish($eventId);
```

Comment:
Is the order important? Should it be the last command?

Reply:
I don't see why it would matter, you can update events regardless they are in `DRAFT` or `READY_FOR_VALIDATION`. The Normal EntryAPI also does it as a first `Command`.